deploy/scripts/configure.sh

```diff
@@ -53,6 +59,20 @@ AKS_CLUSTER_NAME=$(az aks list -g ${PROCESSING_RESOURCE_GROUP} \
     --query "[?tags.type && tags.type == 'k8s'].name" -otsv)
 ACR_DNS=$(az acr list -g ${PROCESSING_RESOURCE_GROUP} \
     --query "[?tags.environment && tags.environment == '$ENV_NAME'].loginServer" -otsv)
+AKS_OIDC_ISSUER="$(az aks show --resource-group ${PROCESSING_RESOURCE_GROUP} \
+    --name ${AKS_CLUSTER_NAME} --query "oidcIssuerProfile.issuerUrl" -o tsv)"
+
+USER_ASSIGNED_CLIENT_ID="$(az identity show -g ${PROCESSING_RESOURCE_GROUP} \
+    --name $UAMI --query 'clientId' -o tsv)"
+IDENTITY_TENANT=$(az aks show --name ${AKS_CLUSTER_NAME} \
+    --resource-group ${PROCESSING_RESOURCE_GROUP} --query identity.tenantId -o tsv)
+
+az identity federated-credential create --name ${FEDERATED_IDENTITY_NAME} \
```

Comment:
This is pretty new and might require an az client update. Not sure if we need to document that somewhere or update instructions. I like the approach, though. (Also see my other comment about workload identity.)

Reply:
Updated the README.md file under deploy to include the additional pre-requisites including the az cli version (2.40.0 or later) to support `az identity federated-credential create` command.